src/pudl/transform/ferc1.py

```diff
@@ -2931,6 +2926,82 @@ def merge_xbrl_metadata(
         return df
 
 
+class AccumulatedDepreciationPlantFerc1TableTransformer(Ferc1AbstractTableTransformer):
+    """Transformer class for :ref:`accumulated_depreciation_plant_ferc1` table."""
+
+    table_id: TableIdFerc1 = TableIdFerc1.ACCUMULATED_DEPRECIATION_PLANT_FERC1
+    has_unique_record_ids: bool = False
+
+    @cache_df("dbf")
+    def process_dbf(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Accumulated Depreciation table specific DBF cleaning operations.
+
+        * Drop functional classification rows.
+        * Drop unstructured data rows.
+        * Set utility_type to electric for all rows.
+        """
+        excluded_labels = [
```

Comment:
Are there other examples of "drop rows with these labels" in other tables that would justify making this into a (simple) parameterized method?

Reply:
Hold off on this until #2155 gets merged.